glib/src/lib.rs

```diff
@@ -88,6 +88,7 @@ pub use once_cell;
 
 pub use glib_macros::{
     clone, gflags, object_interface, object_subclass, Downgrade, GBoxed, GEnum, GErrorDomain,
+    Shared,
```

Comment:
`GSharedBoxed` or similar maybe? Consistent with `GBoxed`, `GEnum`, etc. and it makes it clear that this is still a boxed type